recipes/python-language-server/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "python-language-server" %}
+{% set version = "0.1.2" %}
+{% set hash_type = "sha256" %}
+{% set hash = "7ba71ddcd9e6f008ed37ebe746622e57ce0b61d2142b8f0dedbd84fdb5dd55a7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/palantir/{{ name }}/archive/{{ version }}.tar.gz
```

Comment:
The issue is that the block for `source` should be:

```
fn: {{ version }}.tar.gz
url: https://github.com/palantir/{{ name }}/archive/{{ version }}.tar.gz
sha256: {{ hash }}
```

Currently conda is trying to clone the tarball since you are giving it a `git_url`.